sdk/spring/azure-spring-boot/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
     ```
 - Check scope parameter for AAD authorization requests before configuration. Necessary permissions would be automatically added if needed.
 - Update `com.azure` group id to `com.azure.spring`.
+- Deprecated azure-spring-boot-starter-metrics
```

Comment:
It's called the [azure-spring-boot-metrics-starter](https://mvnrepository.com/artifact/com.microsoft.azure/azure-spring-boot-metrics-starter).